CONTRIBUTING.md

```diff
@@ -1,3 +1,17 @@
 For now just do it :)
 
-Its Highly In-Dev right now anyway ...
\ No newline at end of file
+Its Highly In-Dev right now anyway ...
+
+## Setup DEV Environment
+
+To start working on ArmA.Studio
+
+1. If you have not already, [download](https://www.visualstudio.com/de/downloads/) and install Visual Studio
```

Comment:
Forgot to mention that the /de/ link still redirects to /downloads/, so this isn't necessary. However it may confuse people if they get redirected to a /de/ link if they're not german. Also /downloads/ is the default so it's rather pointless to redirect the user.